data-prepper-script/build.gradle

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+plugins {
+    id 'antlr'
+    id 'java'
+    id 'idea'
+}
+
+group 'org.opensearch.dataprepper'
```

Comment:
Both `group` and `version` should be removed, to clarify @cmanning09 's comment.